README.md

```diff
@@ -921,6 +922,7 @@ http://www.listjs.com
 * [Jaavascript tutorial](https://hackr.io/tutorials/learn-javascript) - Learn Javascript online from a diverse range of user ranked online tutorials.
 
 
+
```

Comment:
why this space?

Reply:
ho,  it might due to my editor markdown formater

Reply:
nice, can you please remove it?

Reply:
sure

Reply:
thank you!